src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/Handlers/MetadataReferenceItemHandler.cs

```diff
@@ -11,11 +11,16 @@ namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices.Handlers
     /// </summary>
     [Export(typeof(ILanguageServiceCommandLineHandler))]
     [AppliesTo(ProjectCapability.CSharpOrVisualBasicLanguageService)]
-    internal class MetadataReferenceFilesLanguageServiceItemHandler : ILanguageServiceCommandLineHandler
+    internal class MetadataReferenceItemHandler : ILanguageServiceCommandLineHandler
     {
+        private readonly UnconfiguredProject _unconfiguredProject;
+
         [ImportingConstructor]
-        public MetadataReferenceFilesLanguageServiceItemHandler(UnconfiguredProject project)
+        public MetadataReferenceItemHandler(UnconfiguredProject project)
         {
+            Requires.NotNull(project, nameof(project));
```

Comment:
The reason I am saying this because, unless `AllowDefault = true` attribute is set, the MEF imported object is expected not to be `null`.